### PR TITLE
Update job descriptions

### PR DESCRIPTION
**Pull request by jnm**

### CODE REVIEW

File: join-our-team/developers.html [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+<style>
+    .container h2 small { text-transform: none; }
+    .section-text-page table { width: 100%; }
+    .section-text-page td {
+        border: none; vertical-align: top; max-width: 33%;
+    }
+    .section-text-page table h3 { margin: 0 !important; }
+</style>
+
+<section class="section section-text-page">
```
Comment (magicznyleszek) [inline join-our-team/developers.html]: Remove above `<style>`. Add an id attribute to the section (`id="join-our-team"`). Create a `scss/_join-our-team.scss` file and nest the styles in an id selector:

```scss
#join-our-team {
    h2 small {text-transform: none;}
    .section-text-page {
        table {
            width: 100%;
            h3 {margin: 0 !important;}
        }
        td {
            border: none;
            vertical-align: top;
            max-width: 33%;
        }
    }
}
```

Comment (magicznyleszek) [reply join-our-team/developers.html]: Oh, and you need to import the new file in `scss/index.scss` (similarly to `"khc"` and others)

File: join-our-team/developers.html [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+        <h3>How to apply</h3>
+
+        <p>We seek contractors who desire a long-term relationship (ideally one
+        year or more) and who can commit significant time to our project (at
+        least twenty hours per week, and full-time is also possible). If your
+        skills and schedule dovetail with our needs, and you are up for the
+        challenging-yet-rewarding work of supporting humanitarians worldwide,
+        <a href="https://ee.humanitarianresponse.info/::d26zTYKn">please fill
+        out this questionnaire to apply</a>.
```
Comment (magicznyleszek) [inline join-our-team/developers.html]: `</p>` missing

File: join-our-team/developers.html (hunk 1 above)
Comment (jnm) [reply join-our-team/developers.html]: :+1:

**Comment by tinok** [comment]
adding updated job descriptions

**Pull request merged**